src/Kafka/CreateKafkaInstanceWithSizes/CreateKafkaInstanceMachine.ts

```diff
@@ -508,10 +508,14 @@ const CreateKafkaInstanceMachine = createMachine(
       noSizes: ({ sizes }) => sizes === undefined,
       emptySizes: ({ sizes }) =>
         sizes !== undefined && sizes.standard.length === 0,
-      sizeIsInQuota: ({ form, capabilities }) =>
-        capabilities !== undefined &&
-        form.size !== undefined &&
-        form.size.quota <= capabilities.remainingQuota,
+      sizeIsInQuota: ({ form, capabilities }) => {
+        if (capabilities === undefined) return false;
+        if (capabilities.plan === "trial") return true;
```

Comment:
app-sevices-ui-shared has plan constants defined. Can we use that module in this module?